README.md

```diff
@@ -10,6 +10,28 @@ The content for the site lives in the `content/` folder. Fork this repo and send
 - Updated course information
 - Anything you can think of
 
+Compiling site locally
+----------------------
+
+Install [Lektor](https://www.getlektor.com/)
+
+```bash
+# Clone the repo
+git https://github.com/jackharrhy/Website.git`
```

Comment:
It looks like there's a `clone` missing here

Reply:
💯 

cbdda82ebacc7d6c97d6312857e0cf3895d2e4f9